### PR TITLE
Stars

### PR DESCRIPTION
**Pull request by edgarlokys**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="block-stars stars--0">
```
Comment (mykhalenych) [inline src/index.html:11]: missed requirement, this block must has class stars stars--0
Screenshot 2023-02-16 at 16 59 10

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,47 @@
...
+      <div class="block-stars stars__star"></div>
```
Comment (mykhalenych) [inline src/index.html:12]: and children's must have stars__star
Screenshot 2023-02-16 at 17 01 38

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
+}
+
+.stars--1 :first-child {
```
Comment (mykhalenych) [inline src/style.css:16]:
```suggestion
.stars--1 :first-child,
.stars--2 :nth-child(-n + 2),
... {
  background-image: url(images/star-active.svg);
}
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+.block-stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
```
Comment (mykhalenych) [inline src/style.css:13]: last child shouldn't has margin-right

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (mykhalenych) [inline src/index.html:11]:
```suggestion
    <div class="stars stars--0">
```

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+.block-stars__stars {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
+}
+
+.block-stars__stars:last-child {
```
Comment (Viktor-Kost) [inline src/style.css:16]: ok, yet its better to use `:not(:last-child)` for similar cases
```
.stars__star:not(:last-child) {
  margin-right: 4px
}
```

File: src/index.html [hunk 6]
```diff
@@ -8,6 +8,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="block-stars__stars"></div>
```
Comment (Viktor-Kost) [inline src/index.html:12]: children should be as an elements of stars block (according to BEM methodology)
And use singular name for you star
Apply all over the project
```suggestion
      <div class="stars__star"></div>
```

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+.stars {
+  display: flex;
+}
+
+.block-stars__stars {
+  height: 16px;
+  width: 16px;
+  background: url(images/star.svg) no-repeat center;
+  margin-right: 4px;
```
Comment (Viktor-Kost) [inline src/style.css:13]:
```suggestion
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]

**Pull request closed without merging**